src/app/components/wrapper-components/online-game-wrapper/online-game-wrapper.quarto.component.spec.ts

```diff
@@ -357,45 +356,42 @@ describe('OnlineGameWrapperComponent of Quarto:', () => {
             lastUpdateTime: serverTimestamp(),
         };
         // TODO: should receive somewhere some kind of Timestamp written by DB
-        expect(partDAO.update).toHaveBeenCalledOnceWith('joinerId', expectedUpdate );
-        tick(wrapper.joiner.maximalMoveDuration * 1000);
+        expect(partDAO.update).toHaveBeenCalledOnceWith('configRoomId', expectedUpdate );
+        tick(wrapper.configRoom.maximalMoveDuration * 1000);
     }));
     it('should forbid making a move when it is not the turn of the player', fakeAsync(async() => {
-        const messageDisplayer: MessageDisplayer = TestBed.inject(MessageDisplayer);
-
         // Given a game
         await prepareStartedGameFor(UserMocks.CREATOR_AUTH_USER);
+        const messageDisplayer: MessageDisplayer = TestBed.inject(MessageDisplayer);
         spyOn(messageDisplayer, 'gameMessage');
         tick(1);
```

Comment:
to be removed too ?

Reply:
la ligne hihglighté définitement pas, on observe une méthode qu'on veut s'assurer d'appeler. Le tick(1), yes.